docs/js/Brackets.js

```diff
@@ -25,6 +25,10 @@ export class Brackets {
         return Brackets.#openBrackets.includes(token)
     }
 
+    static isClosed(token) {
+        return Brackets.#closedBrackets.includes(token)
+    }
```

Comment:
Please add a [JSDoc](https://jsdoc.app/) for this function. You can mostly copy it from `isOpen`.